Dockerfile

```diff
@@ -4,18 +4,19 @@ LABEL maintainer="team@semantic.works"
 
 RUN apt-get update && apt-get -y upgrade && apt-get -y install git openssh-client rsync
 
-ENV MU_SPARQL_ENDPOINT 'http://database:8890/sparql'
-ENV MU_APPLICATION_GRAPH 'http://mu.semte.ch/application'
-ENV NODE_ENV 'production'
+ENV MU_SPARQL_ENDPOINT='http://database:8890/sparql'
```

Comment:
changed this because i got warnings that the old way is now deprecated when i built.

Reply:
Have we lost 2 env vars whilst renaming the structure of one?

Reply:
i don't think so? which ones are missing?